exercises/largest-series-product/examples/src/LargestSeriesProduct.purs

```diff
@@ -0,0 +1,31 @@
+module LargestSeriesProduct
+  ( largestProduct
+  ) where
+
+import Prelude
+import Data.Array (drop, length, take, (:))
+import Data.Foldable (maximum, product)
+import Data.Int (fromString)
+import Data.Maybe (Maybe(..))
+import Data.String (singleton, toCharArray)
+import Data.Traversable (sequence)
+
+spans :: forall a. Int -> Array a -> Array (Array a)
+spans n a
+  | length a < n = []
+  | otherwise    = take n a : spans n (drop 1 a)
+
+toIntArray :: String -> Maybe (Array Int)
+toIntArray = toCharArray
+             >>> map (singleton >>> fromString) 
```

Comment:
Thanks @paf31, missed that one, I'll try to remember :) I've pushed an update https://github.com/icyrockcom/xpurescript/commit/38bcfc6e314141fa96e0c2c97b517ea9c002c628 if you prefer to merge that one.